### DISCUSSION TITLE
Easiest way to check current version?

### DISCUSSION THREAD

**Discussion by kieron** [mbround18/valheim-docker#305] [General]
Hey, as title > what's easiest way to check what version my server is on? For purpose of troubleshooting if I'm inline with current game version.

Thanks

**Comment by mbround18** [comment]
`odin update --check`

It will return with a 0 if no update available and a 10 if an update is available

![image](https://user-images.githubusercontent.com/12646562/112344142-0b1c3580-8c81-11eb-9634-eaf3f3501eda.png)
